src/utils.ts

```diff
@@ -388,7 +400,7 @@ const utils = (module.exports = {
         const newKey = prevKey ? `${prevKey}[${key}]` : key;
 
         if (utils.isObject(value)) {
-          if (!Buffer.isBuffer(value) && !value.hasOwnProperty('data')) {
+          if (!Buffer.isBuffer(value) && !hasOwn(value, 'data')) {
```

Comment:
Can we inline that hasOwn function? I don't see a lot of advantages of that alias.

Reply:
I agree, just removed the hasOwn function.